app/Policies/PostPolicy.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Policies;
+
+use App\Entities\Post;
+use App\Entities\User;
+use App\Libraries\Policies\PolicyInterface;
+
+class PostPolicy implements PolicyInterface
+{
+    /**
+     * Determines if the current user can create a new thread.
+     */
+    public function create(): bool
+    {
+        return true;
```

Comment:
I think my sample file may have been crap. We wouldn't actually want to return `true` for all of these unless we want guests and all other roles to be able to post. I can think of 3 ways to handle this: 

1)  The methods should accept the user instance and check that with standard Shield stuff. 
2) Completely remove the methods that don't have custom checks since they should use the standard Shield call at that point. 
3) Use the `before()` method to do a standard Shield permission check unless it's the `edit` method since that has additional needs. 

I'd opt for #2 as the simplest and cleanest personally.

Reply:
Done.